rusty_json_schema.gemspec

```diff
@@ -5,10 +5,10 @@ require_relative "lib/rusty_json_schema/version"
 Gem::Specification.new do |spec| # rubocop:disable Metrics/BlockLength
   spec.name        = "rusty_json_schema"
   spec.version     = RustyJSONSchema::VERSION
-  spec.authors     = ["Leszek Zalewski"]
+  spec.authors     = ["Leszek Zalewski", "Babbel GmbH"]
   spec.email       = ["leszekzalewski@fastmail.fm"]
```

Comment:
maybe we can change email as well?

Reply:
mmm I'm not sure which email to put, I checked other repos and I didn't find an answer 🤔